### ISSUE TITLE
tests/testthat/cleanup_temp.R should arguably fully clean up /tmp but doesn't

### ISSUE THREAD

**Issue by malcook** [RGLab/flowWorkspace#402]
The tests in tests/testthat/cleanup_temp.R should probably fully delete all the files it creates in the temp directory.

This can be achieved by adding a call to `gs_cleanup_temp(gs_untouched)` at the end

I discovered this while running the tests by hand as part of studying them to understand how these functions worked since I was running out of disk during a long running data-intensive session involving flowWorkspace.

**Comment by mikejiang** [comment]
The test data generated in temp folder in unit test is pretty small, I doubt this was the major source of contribution to your out of disk problem

**Comment by malcook** [comment]
Agreed – this is a minor issue – I nonetheless thought it worth mentioning

~ ***@***.***
